File: app/src/main/java/aws/apps/usbDeviceEnumerator/ui/main/TabViewHolder.java
package aws.apps.usbDeviceEnumerator.ui.main;

import android.view.View;
import android.widget.ListView;
import android.widget.TextView;

import aws.apps.usbDeviceEnumerator.R;
import butterknife.BindView;
import butterknife.ButterKnife;

public class TabViewHolder {

    @BindView(android.R.id.list)
    protected ListView list;

    @BindView(android.R.id.empty)
    protected View empty;

    @BindView(R.id.count)
    protected TextView count;

    public TabViewHolder(final View rootView) {
        ButterKnife.bind(this, rootView);
        list.setEmptyView(empty);
        list.setChoiceMode(ListView.CHOICE_MODE_SINGLE);
    }

    public ListView getList() {
        return list;
    }

    public TextView getCount() {
        return count;
    }
}

File: app/src/main/java/aws/apps/usbDeviceEnumerator/ui/main/tabs/TabViewHolder.kt
package aws.apps.usbDeviceEnumerator.ui.main.tabs;

import android.view.View;
import android.widget.ListView;
import android.widget.TextView;

import aws.apps.usbDeviceEnumerator.R;

public class TabViewHolder {

    private final View rootView;
    private final ListView list;
    private final View empty;
    private final TextView count;

    public TabViewHolder(final View rootView) {
        this.rootView = rootView;

        list = rootView.findViewById(android.R.id.list);
        empty = rootView.findViewById(android.R.id.empty);
        count = rootView.findViewById(R.id.count);

        list.setEmptyView(empty);
        list.setChoiceMode(ListView.CHOICE_MODE_SINGLE);
    }

    public ListView getList() {
        return list;
    }

    public TextView getCount() {
        return count;
    }
}
